pkg/endpoint/directory_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
+
+package endpoint
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/cilium/cilium/pkg/checker"
+	"gopkg.in/check.v1"
+)
+
+func (s *EndpointSuite) TestMoveNewFilesTo(c *check.C) {
+	oldDir := c.MkDir()
+	newDir := c.MkDir()
+	f1, err := ioutil.TempFile(oldDir, "")
+	c.Assert(err, check.IsNil)
+	f2, err := ioutil.TempFile(oldDir, "")
+	c.Assert(err, check.IsNil)
+	f3, err := ioutil.TempFile(newDir, "")
+	c.Assert(err, check.IsNil)
+
+	// Copy the same f4 file in both directories to make sure the same files
```

Comment:
typo?
```suggestion
	// Copy the same file in both directories to make sure the same files
```

Reply:
The comment was already there in `pkg/common` and it looks like it refers to an `f4 *os.File` which never existed. Will fix.